src/app/treasure-hunt/treasure-hunt-calculator-services/electricity-reduction-treasure-hunt.service.ts

```diff
@@ -96,7 +96,7 @@ export class ElectricityReductionTreasureHuntService {
 
   convertElectricityReduction(reduction: ElectricityReductionData, oldSettings: Settings, newSettings: Settings): ElectricityReductionData {
     //imperial: hp, metric: kW
-    reduction.nameplateData.ratedMotorPower = this.convertUnitsService.convertPowerValue(reduction.nameplateData.ratedMotorPower, oldSettings, newSettings);
+    //reduction.nameplateData.ratedMotorPower = this.convertUnitsService.convertPowerValue(reduction.nameplateData.ratedMotorPower, oldSettings, newSettings);
```

Comment:
If there's no logic to convert, this method doesn't need to be called. Can remove this and any other conversion logic not being used